src/vdaf/prio3.rs

```diff
@@ -562,37 +575,70 @@ where
     }
 }
 
-/// Message sent by the [`Client`](crate::vdaf::Client) to each
-/// [`Aggregator`](crate::vdaf::Aggregator) during the Sharding phase.
+/// Message broadcast by the [`Client`] to every [`Aggregator`] during the Sharding phase.
+#[derive(Clone, Debug, PartialEq, Eq)]
+pub struct Prio3PublicShare<const L: usize> {
+    /// Contributions to the joint randomness from every aggregator's share.
+    joint_rand_parts: Vec<Seed<L>>,
```

Comment:
nit: An `Option` would be slightly more aligned with semantics of optional fields elsewhere in this code. Unless you have reason to think has a hidden cost, I would suggest using `Option` her for consistency.
```suggestion
    joint_rand_parts: Option<Vec<Seed<L>>>,
```